Minesweeper.py

```diff
@@ -89,7 +89,7 @@ def game_loop():
             player_pos = new_pos
             moves += 1
 
-        if moves == BOARD_SIZE * BOARD_SIZE - NUM_MINES:
+        if moves == BOARD_SIZE * BOARD_SIZE - NUM_MINES + 1:  # Error: should not have +1
```

Comment:
There's an error in the condition for checking if the player has won. The `+1` should be removed.
```diff
-        if moves == BOARD_SIZE * BOARD_SIZE - NUM_MINES + 1:  # Error: should not have +1
+        if moves == BOARD_SIZE * BOARD_SIZE - NUM_MINES:  # Error: should not have +1
```